Client/Frontend/Brave Rewards/Ads/AdsNotificationHandler.swift

```diff
@@ -89,8 +105,24 @@ public class AdsNotificationHandler: BraveAdsNotificationHandler {
     })
   }
   
+  // This method can be used to display a wallet connection prompt when we detect users are visiting
+  // a web3 site.
+  public func showWalletConnectionNotification() {
+    let walletNotification = AdNotification.customAd(title: "", body: "", url: AdNotification.walletNotificationURL)
```

Comment:
We build this `AdNotification.customAd()` with a empty title/body twice. Might make sense to use a static let on `AdNotification` to define a wallet connection ad with the static `AdNotification.walletNotificationURL`?

Reply:
This is completed redesigned. All the notifications now conform a `BraveNotification` protocol